fix(curriculum): Recipe Tracker improvements (#57694)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5635.md

You can now fill in each item of the `recipes` array with values for the `averageRating`, `totalIngredients`, and `difficultyLevel` properties.

For now, access the `averageRating`, `totalIngredients`, and `difficultyLevel` of `recipe1` and set them to the appropriate results of function calls and arguments.

# --hints--

You should access the `averageRating` property of `recipe1`.

```js
assert.isNotNull(recipe1.averageRating);
```

> You should set the `averageRating` of `recipe1` you accessed to the result of `getAverageRating` with `recipe1.ratings` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getAverageRating` with `recipe1.ratings` to the `averageRating` property of `recipe1`.
```

> You should set the `averageRating` of `recipe1` you accessed to the result of `getAverageRating` with `recipe1.ratings` as its parameter.

Reviewer 2: ```suggestion
You should set the `averageRating` of `recipe1` you accessed to the result of `getAverageRating` with `recipe1.ratings` as its argument.
```

```js
assert.strictEqual(recipe1.averageRating, getAverageRating(recipe1.ratings));
```

You should access the `totalIngredients` property of `recipe1`.

```js
assert.isNotNull(recipe1.totalIngredients);
```

> You should set the `totalIngredients` of `recipe1` you accessed to the result of `getTotalIngredients` with `recipe1.ingredients` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getTotalIngredients` with `recipe1.ingredients` to the `totalIngredients` property of `recipe1`.
```

> You should set the `totalIngredients` of `recipe1` you accessed to the result of `getTotalIngredients` with `recipe1.ingredients` as its parameter.

Reviewer 2: ```suggestion
You should set the `totalIngredients` of `recipe1` you accessed to the result of `getTotalIngredients` with `recipe1.ingredients` as its argument.
```

```js
assert.strictEqual(recipe1.totalIngredients, getTotalIngredients(recipe1.ingredients));
```

You should access the `difficultyLevel` property of `recipe1`.

```js
assert.isNotEmpty(recipe1.difficultyLevel);
```

> You should set the `difficultyLevel` of `recipe1` you accessed to the result of `getDifficultyLevel` with `recipe1.cookingTime` as its argument.

Reviewer: ```suggestion
You should assign the result of calling `getDifficultyLevel` with `recipe1.cookingTime` to the `cookingTime` property of `recipe1`.
```

> You should set the `difficultyLevel` of `recipe1` you accessed to the result of `getDifficultyLevel` with `recipe1.cookingTime` as its parameter.

Reviewer 2: ```suggestion
You should set the `difficultyLevel` of `recipe1` you accessed to the result of `getDifficultyLevel` with `recipe1.cookingTime` as its argument.
```
